src/components/UI/navigation/index.js

```diff
@@ -133,7 +133,7 @@ class Navigation extends React.Component {
             </NavLink>
             {this.props.showAppCatalog && (
               <NavLink activeClassName='active' to='/app-catalogs/'>
-                Apps
+                App Catalogs
```

Comment:
I noticed there was a mental confusion when going between the Apps Tab on the cluster detail page and the Apps Link in the navigation. This resolves that and makes it clearer what you'll see when you click the link in the navigation.